Makefile

```diff
@@ -170,18 +169,15 @@ endif
 
 c_tests:
 	set -e ;\
-	cd src/tests ;\
-	find $(abspath $(BINROOT)/src/tests) -name "test_*" -type f -executable -exec ${GDB_CMD} {} \;
+	find $(abspath $(BINROOT)/tests/ctests) -name "test_*" -type f -executable -exec ${GDB_CMD} {} \;
 
 cpp_tests:
 ifeq ($(TEST),)
 	set -e ;\
-	cd src/tests ;\
-	$(GDB_CMD) $(abspath $(BINROOT)/src/cpptests/rstest)
+	find $(abspath $(BINROOT)/tests/cpptests) -name "test_*" -type f -executable -exec ${GDB_CMD} {} \;
 else
 	set -e ;\
-	cd src/tests ;\
-	$(GDB_CMD) $(abspath $(BINROOT)/src/cpptests/rstest) --gtest_filter=$(TEST)
+	$(GDB_CMD) $(abspath $(BINROOT)/tests/cpptests/$(TEST))
```

Comment:
Can we keep the `--gtest_filter=$(TEST)` argument?

Reply:
:+1: solved in the last commit.